### PR TITLE
Added Heliarch first contact scenario.

### PR DESCRIPTION
**Pull request by Lineth**
For those who land on the ring of friendship first.

### CODE REVIEW

**Comment by jafdy** [comment]
This would let the normal missions trigger when you land on a coalition planet for the first time even after triggering this mission.

File: data/coalition missions.txt [hunk 1]
```diff
@@ -79,7 +79,57 @@ mission "Coalition: First Contact"
...
+		government "Heliarch"
+	to offer
+		not "Coalition: First Contact: offered"
+	on offer
+		"reputation: Coalition" += 20
+		"reputation: Heliarch" += 20
+		conversation
+			`Your first view of the landing area confirms beyond a doubt that this is a Quarg ringworld, not just an imitation of one. The interior architecture is unmistakable. But no Quarg are present here, only three alien races you are not familiar with. First are giant beetles, about a meter tall, who scurry quickly toward your ship and swarm around it, gawking at you, as soon as you land. The second species, even more disturbing than the beetles, are giant spiders. And the third species are centaurs. Not just something vaguely resembling centaurs; they look like they could have stepped right off the page of a story book from ancient Earth.`
+			`	Soon, a delagation consisting of one of each of these races approaches you. The circlets seem to be not just a badge of office, but seem to contain electronics as well. `
+			`	The centaur speaks first. "We, the Heliarchs, who drove out the oppressors, who maintain peace, we the human visitor greet."`
```
Comment (Bladewood) [inline data/coalition missions.txt]: Contains "we" twice?

File: data/coalition missions.txt [hunk 2]
```diff
@@ -79,7 +79,57 @@ mission "Coalition: First Contact"
...
+	source
+		government "Heliarch"
+	to offer
+		not "Coalition: First Contact: offered"
+	on offer
+		"reputation: Coalition" += 20
+		"reputation: Heliarch" += 20
+		conversation
+			`Your first view of the landing area confirms beyond a doubt that this is a Quarg ringworld, not just an imitation of one. The interior architecture is unmistakable. But no Quarg are present here, only three alien races you are not familiar with. First are giant beetles, about a meter tall, who scurry quickly toward your ship and swarm around it, gawking at you, as soon as you land. The second species, even more disturbing than the beetles, are giant spiders. And the third species are centaurs. Not just something vaguely resembling centaurs; they look like they could have stepped right off the page of a story book from ancient Earth.`
+			`	Soon, a delagation consisting of one of each of these races approaches you. The circlets seem to be not just a badge of office, but seem to contain electronics as well. `
```
Comment (Bladewood) [inline data/coalition missions.txt]: delegation*

**Comment by Lineth** [comment]
My apologies for the bugs. Was near midnight when I made this and is my first serious attempt at a mission.

**Comment by KiLEdEnNis** [comment]
coming along nicely though, keep it up

**Comment by whismerhill** [comment]
am I misreading something or does this quest actually adds +20 rep toward heliarchs & coalition ?

**Comment by Bladewood** [comment]
It does, and that's the way it should be.
The original First Contact mission adds 10 reputation for each once you are offered it by landing on a Coalition planet, and you get another 10 reputation for each when you complete the mission by landing on a Heliarch ringworld.

**Comment by Lineth** [comment]
Closed because of [2ab0555](https://github.com/endless-sky/endless-sky/commit/2ab0555277ae0e12c5fa4e867821c3f8144e6c6a).

**Pull request closed without merging**